ssc/elections/2023H2/MOHAMED_OMAR.md

```diff
@@ -0,0 +1,10 @@
+# Mohamed Omar
+Mohamed Omar leads SPIFFE/SPIRE efforts at Confluent.
+He has participated in the SPIFFE community for over 2 years and would be an excellent member of the steering committee.
+
+## Reference
+**Email:** momar@confluent.io
+**GitHub:** https://github.com/moemar
```

Comment:
I believe it should be https://github.com/moe-omar, this is pointing to a different person on GitHub.